Detector/DetFCChhECalDigital/src/ECalBarrel_geo.cpp

```diff
@@ -0,0 +1,356 @@
+// DD4hep includes
+#include "DD4hep/DetFactoryHelper.h"
+#include "DD4hep/Shapes.h"
+#include "DD4hep/Objects.h"
+
+#include "XML/XMLElements.h"
+
+// Gaudi
+#include "GaudiKernel/ServiceHandle.h"
+#include "GaudiKernel/IMessageSvc.h"
+#include "GaudiKernel/MsgStream.h"
+
+#include "cmath"
+
+
+namespace det {
+
+static dd4hep::detail::Ref_t createECal (dd4hep::Detector& lcdd,dd4hep::xml::Handle_t xmlElement,
```

Comment:
This name is too generic (all factories are under same `det` namespace). Can you add `Digital`? Change should be also done in the declaration at the end of that file.